RemoteControl/RemoteControl.json

```diff
@@ -180,6 +178,59 @@
             "type": "integer",
             "example": 1
         },
+        "macAddress": {
+            "summary": "The MAC address of the remote in hex-colon format",
+            "type": "string",
+            "example": "E8:1C:FD:9A:07:1E"
+        },
+        "sessionId": {
```

Comment:
Should this be part of the firmwareUpdateStatus object? If not, we use sessionIds for other constructs, may be worth signifying that this session id is for fw update in the name

Reply:
I don't see any problem with adding the mac address to the firmwareUpdateStatus object...

Reply:
Misunderstood, no comment